.github/workflows/pulsar-ci.yaml

```diff
@@ -466,6 +465,68 @@ jobs:
         run: |
           gh-actions-artifact-client.js delete pulsar-java-test-image.zst
 
+  cpp-tests:
+    name:
```

Comment:
```suggestion
    name: "CI - CPP, Python Tests"
```

Reply:
See this conversation https://github.com/apache/pulsar/pull/17571#discussion_r967208132 
I will do a follow up pr